Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:10-alpine
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache yarn
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ENV HOST 0.0.0.0
+
+ARG BRANCH
+ENV NODE_ENV $BRANCH
+
+RUN cd /usr/src/app/
+COPY . .
+RUN yarn
+RUN cd components/AddToCalendar/src && npm install --production
+RUN cd components/Calender && npm install --production
+RUN cd components/conference-cal && npm install --production
+
+
+
+RUN yarn build:ci
+RUN rm -rf node_modules
+RUN npm install --production
```

Comment:
are you using `yarn` or `npm`.

Reply:
both, yarn does not do a nice production install. It is a giant for some reason.

Reply:
build with dev dependency node modules , delete then build with production deps.